packages/manager/apps/ips/src/data/api/get/dedicatedServerVmac.ts

```diff
@@ -29,3 +29,17 @@ export const getdedicatedServerVmac = async (
     )}/virtualMac`,
     page: 1,
   });
+
+export type ServerModelsType = {
+  data: any;
+};
+
+export const getServerModelsQueryKey = () => [`get/dedicated/server.json`];
+
+export const getServerModels = async (): Promise<IcebergFetchResultV6<
+  ServerModelsType
+>> =>
+  fetchIcebergV6<ServerModelsType>({
+    route: `/dedicated/server.json`,
+    page: 1,
+  });
```

Comment:
should we fetch with iceberg if it's only 1 page / file ?

Reply:
I need the models from this API call so I went with iceberg